<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Andrey Rijov</author>
 <class>Settings_Window</class>
 <widget class="QDialog" name="Settings_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/configure.png</normaloff>:/configure.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="Label_General">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Basic Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="Label_VM_Folder">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>AQEMU VM Folder:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Edit_VM_Folder"/>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="TB_VM_Folder">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/open-folder.png</normaloff>:/open-folder.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Label_Default_VM_Template">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Default VM Template:</string>
       </property>
       <property name="buddy">
        <cstring>CB_Default_VM_Template</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="CB_Default_VM_Template">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="Button_Create_Template_from_VM">
       <property name="toolTip">
        <string>Create Template</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/add.png</normaloff>:/add.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="Label_View">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="Label_Language">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Interface Language:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="CB_Language">
       <item>
        <property name="text">
         <string>English</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Label_Icons_Theme">
       <property name="text">
        <string>GUI Icons Theme:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="CB_Icons_Theme">
       <item>
        <property name="text">
         <string>Crystal SVG (KDE 3)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Oxygen (KDE 4)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Label_VM_Icons_Size">
       <property name="text">
        <string>VM Icon Size:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="CB_VM_Icons_Size">
       <property name="currentIndex">
        <number>4</number>
       </property>
       <item>
        <property name="text">
         <string>16x16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24x24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32x32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>48x48</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64x64</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="CH_Screenshot_for_OS_Logo">
       <property name="text">
        <string>Use Screenshot for OS Icon in Save Mode</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QCheckBox" name="CH_Use_Device_Manager">
       <property name="text">
        <string>Use Device Manager by Default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="Label_Emulator_Control_Settings">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Emulator Control Show Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_23">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QRadioButton" name="RB_Emul_Show_Window">
       <property name="text">
        <string>Show In Separate Window</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RB_Emul_Show_VNC">
       <property name="text">
        <string>Show Window With Embedded Display</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RB_Emul_Show_In_Main_Window">
       <property name="text">
        <string>Show In AQEMU Main Window</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RB_Emul_Show_VNC_In_Main_Window">
       <property name="text">
        <string>Show In AQEMU Main Window With Embedded Display</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RB_Emul_No_Show">
       <property name="text">
        <string>Don't show</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Button_OK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/ok.png</normaloff>:/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/cancel.png</normaloff>:/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="8"/>
 <tabstops>
  <tabstop>CB_Default_VM_Template</tabstop>
  <tabstop>Button_Create_Template_from_VM</tabstop>
  <tabstop>Edit_VM_Folder</tabstop>
  <tabstop>TB_VM_Folder</tabstop>
  <tabstop>CB_Language</tabstop>
  <tabstop>CB_Icons_Theme</tabstop>
  <tabstop>CH_Use_Device_Manager</tabstop>
  <tabstop>CH_Screenshot_for_OS_Logo</tabstop>
  <tabstop>RB_Emul_Show_Window</tabstop>
  <tabstop>RB_Emul_Show_VNC</tabstop>
  <tabstop>RB_Emul_Show_In_Main_Window</tabstop>
  <tabstop>RB_Emul_Show_VNC_In_Main_Window</tabstop>
  <tabstop>RB_Emul_No_Show</tabstop>
  <tabstop>Button_OK</tabstop>
  <tabstop>Button_Cancel</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Button_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>Settings_Window</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
